src/assets/src/components/search.tsx

```diff
@@ -9,22 +9,25 @@ import { redirectToLogin } from "../utils";
 import { PageProps } from "./page";
 
 
-interface SearchPageParams {
-    term: string;
-}
-
-export function SearchPage(props: PageProps<SearchPageParams>) {
+export function SearchPage(props: PageProps) {
     if (!props.user) {
         redirectToLogin(props.loginUrl);
     }
-    const term = props.match.params.term;
+    let term = (new URLSearchParams(props.location.search)).get('term') ?? undefined;
+    if (term !== undefined) {
+        term = decodeURIComponent(term);
+    }
```

Comment:
I'm not really sure this is necessary anymore. I think since I changed this to a search query and am using `URLSearchParams`, things seem to be decoded already before this logic.

Reply:
Yeah, seems like `URLSearchParams` does decode: https://stackoverflow.com/a/45516812